### PR TITLE
Player setup

### PR DESCRIPTION
**Pull request by vondojr**
Function to create x amount of players and append them to a player list.

### CODE REVIEW

File: source/Player_Setup.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+
+def Player_Setup(Playerlist):
+
+    #Finding out number of players with an error for invalid inputs
+    NumOfPlayers = input("How many people are playing? ")
+
+
+    #A loop for creating various players with name and money amounts
+    try:
```
Comment (JHerren726) [inline source/Player_Setup.py:11]: I like this loop! It's clean and concise.

File: source/Player_Setup.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
+from CardGames import *
```
Comment (JHerren726) [inline source/Player_Setup.py:1]: Great job!

**Review by JHerren726 — APPROVED** [review]
Everything looks great to me! Approved

File: source/Player_Setup.py (hunk 1 above)
Comment (andrewskerratt) [reply source/Player_Setup.py:11]: les go Joooe. Everythn looks good

**Review by andrewskerratt — COMMENTED** [review]
Everything looks good!

**Review by andrewskerratt — APPROVED** [review]

**Pull request merged**